content/docs/1.8.0/advanced-resources/backing-image/backing-image-backup.md

```diff
@@ -34,25 +34,29 @@ Example of YAML code used to create a backup of the sample backing image:
 apiVersion: longhorn.io/v1beta2
 kind: BackupBackingImage
 metadata:
-  name: parrot
+  name: parrot-backup
   namespace: longhorn-system
 spec:
+  backingImage: parrot
+  backupTargetName: default
   userCreated: true
   labels:
     usecase: test
     type: raw
 ```
 
 > **IMPORTANT:**
-> - `name`: Use the same name for the backing image and its backup. If the names are not identical, Longhorn will not be able to find the backing image.
+> - `name`: If the names are not identical, Longhorn will not be able to create a backup of the backing image.
```

Comment:
Is this still valid? the names should be identical.

Reply:
Modified. The name should be unique as the K8s resource convention.